dev/io.openliberty.checkpoint_fat/.classpath

```diff
@@ -3,30 +3,24 @@
 	<classpathentry kind="src" path="fat/src"/>
 	<classpathentry kind="src" path="test-applications/JSPWithEL/src"/>
 	<classpathentry kind="src" path="test-applications/appsecurity/src"/>
-	<classpathentry kind="src" path="test-applications/inventory/src/java"/>
```

Comment:
Each `<classpathentry/>` that gets removed here I would expect to get removed from the `src:` headers in `dev/io.openliberty.checkpoint_fat/bnd.bnd`